docs/operating-scylla/nodetool-commands/compact.rst

```diff
@@ -15,27 +15,11 @@ Syntax
 -------
 .. code-block:: console
 
-   nodetool [options] compact [--partition <partition_key>] [<keyspace> [<cfnames>]...]
+   nodetool [options] compact [<keyspace> [<cfnames>]...]
 
 Options
 --------
 
-* ``-h <host>`` or  ``--host <host>`` - Node hostname or IP address.
-
-* ``-p <port>`` or ``--port <port>`` - Remote JMX agent port number.
-
-* ``--partition <partition_key>`` - String representation of the partition key.
-
-* ``-pp`` or ``--print-port`` - Operate in 4.0 mode with hosts disambiguated by port number.
-
-* ``-pw <password>`` or ``--password <password>`` - Remote JMX agent password.
-
-* ``-pwf <passwordFilePath>`` or ``--password-file <passwordFilePath>`` - Path to the JMX password file.
-
-* ``-u <username>`` or ``--username <username>`` - Remote JMX agent username.
-
-* ``--`` - Separates command-line options from the list of argument(useful when an argument might be mistaken for a command-line option).
```

Comment:
I don't think it's a good idea to remove all these options. People will not be reading a "nodetool book" cover-to-cover, but rather reaching this "nodetool compact" document from a Google (or other) search, and will expect to find a self-contained list of all the options - both "-h" and "--" for example are fairly important. I think we should either keep this list, or move it to a separate page of "common nodetool options" and link to that page. I tend to think the former is cleaner for the user, although risks inconsistencies if we forget to update all pages at a time.

In any case, whether or not you keep this list here or in a separate file, we should definitely remove from the list options that don't make any sense. For example, I don't know why "--partition" was listed here. I also don't know what "-pp" means.

Reply:
I think we should move these to a separate page on common nodetool options. Note that this is the only command I know of which lists these common options. So we should either list these common options on all nodetool commands or none. I think the latter makes more sense. Note that `nodetool help` will also list all these common options.
BTW, our `-h` option doesn't even work at all, see https://github.com/scylladb/scylla-tools-java/pull/342.